### DISCUSSION TITLE
What's the purpose of the mount/unmount methods of secure fields?

### DISCUSSION THREAD

**Discussion by ihojmanb** [mercadopago/sdk-js#103] [Q&A] [answered]
I was wondering what was the real purpose of the `unmount` method (and its counterpart, `mount`) of the secure fields. I'm actually using it to fix a bug, and I think what it does is take away the iframe from my html, but not sure if that's what's really doing.

**Comment by icaldana** [comment, marked as answer]
Hi @ihojmanb, how are you?

**You got it right!** 

- In the [documentation](https://www.mercadopago.cl/developers/en/docs/checkout-api/integration-configuration/card/integrate-via-core-methods#editor_4:~:text=Copiar-,Initialize%20card%20fields,-After%20adding%20the) we can see what `mount`  creates the secure fields structure. 
- On the opposite hand, `unmount` removes the iframes and related event listeners from DOM for scenarios on which you must/want to have more control over the DOM elements.

May I ask you which is the bug you're referring to? Is it related to sdk-js?

**Reply by ihojmanb** [reply]
Hi @icaldana! 

the bug is not related to sdk-js. I needed to unmount the secure fields in order to reload the css I applied to them. It solved my issue.

thanks for your answer, it was very helpful! 😁
